docs: Add note on resource-led policy design in `Best practices` section (#1423)
cerbos/cerbos: docs/modules/policies/pages/best_practices.adoc

This would suit scenarios where a principal can perform _nearly_ every action, and you want to explicitly list disallowed actions.

=== Attribute-led

Consider the following hypothetical scenario:

****

> An organization models it's resources as specific _data sets_. Each data set is unique, as are the principals trying to access them. The organization uses JWTs extensively to manage and transmit identity/contextual information. The resource policies map 1:1 to each data set, and access is governed by arbitrary information (in this case, passed within the JWT).

Reviewer: ```suggestion
An organization models its resources as specific _data sets_. Each data set is unique, as are the principals trying to access them. The organization uses JWTs extensively to manage and transmit identity/contextual information. The resource policies map 1:1 to each data set, and access is governed by arbitrary information (in this case, passed within the JWT).
```

****

> Given the changeable nature of the accessing audiences, it doesn't necessarily make sense to rely on roles for access. What we could do instead is to globally allow all roles and actions and then determine access based on attributes passed in the JWT. Take a look at the following example policy:

Reviewer: ```suggestion
Given the dynamic nature of audiences, it's not practical to enumerate all roles that have access. What we could do instead is to globally allow all roles and actions and then determine access based on attributes passed in the JWT. Take a look at the following example policy:
```
